packages/ast-spec/package.json

```diff
@@ -29,7 +29,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc -b tsconfig.build.json && api-extractor run --local",
+    "build": "tsc -b tsconfig.build.json",
```

Comment:
`api-extractor` seems to be not working with 4.7-beta (ref: https://github.com/microsoft/rushstack/issues/3349). So I've removed `api-extractor` step temporary.

Reply:
fixed by https://github.com/microsoft/rushstack/pull/3394